acoustid/future/fpindex/client.py

```
@@ -79,6 +79,40 @@ class FingerprintIndexClientError(Exception):
+                    id=id,
+                )
+            )
+        )
+
+    def set_attribute(self, name: str, value: int) -> None:
+        self.changes.append(
+            Change(
+                set_attribute=SetAttribute(
+                    name=name,
+                    value=value,
+                )
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)